docs/src/remote-development.md

```diff
@@ -12,57 +12,57 @@ Currently the two instances connect via Zed's servers, but we intend to build pe
-* `doctl compute ssh example-droplet` to connect to a digital ocean droplet
-* `gcloud compute ssh` for a google cloud instance
+- `user@host` will assume you meant `ssh user@host`
+- `ssh -J jump target` to connect via a jump-host
+- `gh cs ssh -c example-codespace` to connect to a GitHub codespace
+- `doctl compute ssh example-droplet` to connect to a DigitalOcean Droplet
+- `gcloud compute ssh` for a Google Cloud instance
 
 ### zed --dev-server-token isn't connecting
 
 There are a few likely causes of failure:
 
-* `zed --dev-server-token` runs but outputs nothing. This is probably because the zed background process is crashing on startup. Try running `zed --dev-server-token XX --foreground` to see any output, and [file a bug](https://github.com/zed-industries/zed) so we can debug it together.
-* `zed --dev-server-token` outputs something like "Connection refused" or "Unauthorized" and immediately exits. This is likely due to issues making outbound HTTP requests to https://collab.zed.dev from your host. You can try to debug this with `curl https://collab.zed.dev`, but we have seen cases where curl is whitelisted, but other binaries are not allowed network access.
-* `zed --dev-server-token` outputs "Zed is already running". If you are editing an existing server, it is possible that clicking "Connect" a second time will work, but if not you will have to manually log into the server and kill the zed process.
+- `zed --dev-server-token` runs but outputs nothing. This is probably because the Zed background process is crashing on startup. Try running `zed --dev-server-token XX --foreground` to see any output, and [file a bug](https://github.com/zed-industries/zed) so we can debug it together.
+- `zed --dev-server-token` outputs something like "Connection refused" or "Unauthorized" and immediately exits. This is likely due to issues making outbound HTTP requests to https://collab.zed.dev from your host. You can try to debug this with `curl https://collab.zed.dev`, but we have seen cases where curl is whitelisted, but other binaries are not allowed network access.
+- `zed --dev-server-token` outputs "Zed is already running". If you are editing an existing server, it is possible that clicking "Connect" a second time will work, but if not you will have to manually log into the server and kill the Zed process.
 
 ## Supported platforms
 
-The remote machine must be able to run Zed. The following platforms should work, though note that we have not exhaustively tested every linux distribution:
+The remote machine must be able to run Zed. The following platforms should work, though note that we have not exhaustively tested every Linux distribution:
 
-* macOS Catalina or later (Intel or Apple Silicon))
-* Linux (x86_64 only). You must have `glibc` installed at version 2.29 (released in 2019) or greater and available globally.
-* Windows is not yet supported.
+- macOS Catalina or later (Intel or Apple Silicon)
```

Comment:
> +

So there's yet another separator for list items?
We use `-` extensively, and I see this for the first time, so not sure that is a good change in this repo.

Reply:
Yeah, it seems like you can do bulleted lists in Markdown using both a hyphen and an asterisk (I didn't know! 😅). Changed to a hyphen because that's what I saw being used on other pages and it's also generally more common. The end, visual result is the same, though! In any case, happy to revert—lmk!